src/libostree/ostree-repo.c

```diff
@@ -3509,16 +3512,57 @@ reload_sysroot_config (OstreeRepo          *self,
    * https://github.com/ostreedev/ostree/issues/1719
    * https://github.com/ostreedev/ostree/issues/1801
    */
+  gboolean valid_bootloader;
   for (int i = 0; CFG_SYSROOT_BOOTLOADER_OPTS_STR[i]; i++)
     {
       if (g_str_equal (bootloader, CFG_SYSROOT_BOOTLOADER_OPTS_STR[i]))
         {
           self->bootloader = (OstreeCfgSysrootBootloaderOpt) i;
-          return TRUE;
+          valid_bootloader = TRUE;
+        }
+    }
+  if (!valid_bootloader) 
+  {
+    return glnx_throw (error, "Invalid bootloader configuration: '%s'", bootloader);
+  }
```

Comment:
Can drop the braces here (which are off indentation)

Reply:
This one looks like it wasn't fixed, but again, let's do https://github.com/ostreedev/ostree/pull/2565 at some point.